diff --git a/dist/utils/validation.cjs b/dist/utils/validation.cjs
index e1ad589da95fbce5c9421ec1e9a35526681c8d5e..90d33a7143a293b3d8e730c103e07cb29e16753b 100644
--- a/dist/utils/validation.cjs
+++ b/dist/utils/validation.cjs
@@ -47,11 +47,6 @@ async function validateTransactionOrigin({ data, from, internalAccounts, origin,
     if (authorizationList || envelopeType === types_1.TransactionEnvelopeType.setCode) {
         throw rpc_errors_1.rpcErrors.invalidParams('External EIP-7702 transactions are not supported');
     }
-    const hasData = Boolean(data && data !== '0x');
-    if (hasData &&
-        internalAccounts?.some((account) => account.toLowerCase() === to?.toLowerCase())) {
-        throw rpc_errors_1.rpcErrors.invalidParams('External transactions to internal accounts cannot include data');
-    }
 }
 exports.validateTransactionOrigin = validateTransactionOrigin;
 /**
@@ -189,17 +184,6 @@ exports.validateParamTo = validateParamTo;
 function validateBatchRequest({ internalAccounts, request, sizeLimit, }) {
     const { origin } = request;
     const isExternal = origin && origin !== controller_utils_1.ORIGIN_METAMASK;
-    const internalAccountsNormalized = internalAccounts.map((account) => account.toLowerCase());
-    if (isExternal &&
-        request.transactions.some((nestedTransaction) => {
-            const normalizedCallTo = nestedTransaction.params.to?.toLowerCase();
-            const callData = nestedTransaction.params.data;
-            const isInternalAccount = internalAccountsNormalized.includes(normalizedCallTo);
-            const hasData = Boolean(callData && callData !== '0x');
-            return isInternalAccount && hasData;
-        })) {
-        throw rpc_errors_1.rpcErrors.invalidParams('External calls to internal accounts cannot include data');
-    }
     if (isExternal && request.transactions.length > sizeLimit) {
         throw new rpc_errors_1.JsonRpcError(ErrorCode.BundleTooLarge, `Batch size cannot exceed ${sizeLimit}. got: ${request.transactions.length}`);
     }
diff --git a/dist/utils/validation.mjs b/dist/utils/validation.mjs
index e61c079e4b5be6a75e58fc73f3e030786b384e1b..0ee7d0d7a9b46ca55590038d122d7a6e075f4cba 100644
--- a/dist/utils/validation.mjs
+++ b/dist/utils/validation.mjs
@@ -44,11 +44,6 @@ export async function validateTransactionOrigin({ data, from, internalAccounts,
     if (authorizationList || envelopeType === TransactionEnvelopeType.setCode) {
         throw rpcErrors.invalidParams('External EIP-7702 transactions are not supported');
     }
-    const hasData = Boolean(data && data !== '0x');
-    if (hasData &&
-        internalAccounts?.some((account) => account.toLowerCase() === to?.toLowerCase())) {
-        throw rpcErrors.invalidParams('External transactions to internal accounts cannot include data');
-    }
 }
 /**
  * Validates the transaction params for required properties and throws in
@@ -183,17 +178,6 @@ export function validateParamTo(to) {
 export function validateBatchRequest({ internalAccounts, request, sizeLimit, }) {
     const { origin } = request;
     const isExternal = origin && origin !== ORIGIN_METAMASK;
-    const internalAccountsNormalized = internalAccounts.map((account) => account.toLowerCase());
-    if (isExternal &&
-        request.transactions.some((nestedTransaction) => {
-            const normalizedCallTo = nestedTransaction.params.to?.toLowerCase();
-            const callData = nestedTransaction.params.data;
-            const isInternalAccount = internalAccountsNormalized.includes(normalizedCallTo);
-            const hasData = Boolean(callData && callData !== '0x');
-            return isInternalAccount && hasData;
-        })) {
-        throw rpcErrors.invalidParams('External calls to internal accounts cannot include data');
-    }
     if (isExternal && request.transactions.length > sizeLimit) {
         throw new JsonRpcError(ErrorCode.BundleTooLarge, `Batch size cannot exceed ${sizeLimit}. got: ${request.transactions.length}`);
     }
